code/modules/mob/living/carbon/human/ego_gifts.dm

```diff
@@ -174,6 +224,63 @@
 	justice_bonus = 10
 	slot = MOUTH_2
 
+/datum/ego_gifts/oppression
+	name = "Oppression"
+	icon_state = "oppression"
+	prudence_bonus = 2
+	justice_bonus = 2
+	slot = MOUTH_1
+
+/datum/ego_gifts/prank
+	name = "Funny Prank"
+	icon_state = "prank"
+	prudence_bonus = 4
+	slot = HELMET
+
+/datum/ego_gifts/desire
+	name = "Sanguine Desire"
+	icon_state = "desire"
+	fortitude_bonus = 4
+	slot = MOUTH_2
+
+/datum/ego_gifts/thosewhoknowthecrueltyofwinterandthearomaofroses
```

Comment:
The hell is this name.

Reply:
That is the ACCURATE NAME of the EGO GIFT. It is NOT Frost Splinter for WHATEVER GOD FORSAKEN REASON.
![image](https://user-images.githubusercontent.com/61567407/187319901-974296a5-72e4-4a13-9173-2645bc06a509.png)